docs/semgrep-assistant/overview.md

```diff
@@ -49,42 +49,48 @@ By categorizing your code through component tags, Semgrep Assistant can help you
 
 ![Semgrep Assistant Component tag list](/img/assistant-component-tags.png)
 
+Component tags are available in SCP's **Findings** page.
+
 ### Auto-triage
 
-Semgrep Assistant uses GPT-4's understanding of programming languages and libraries and your code and triage history to auto-triage findings and indicate if a finding can safely be ignored. For every recommendation to ignore a finding, Semgrep also provides guidance with an explanation on why this is the case.
+Semgrep Assistant uses GPT-4's understanding of programming languages and libraries, and your code and triage history, to auto-triage findings and suggest whether a finding can safely be ignored. For every recommendation to ignore a finding, Semgrep also provides guidance with an explanation on why this is the case.
 
-Semgrep presents these recommendations through Slack and PR or MR comments, so developers can fix or triage an issue without switching contexts.
+Auto-triage recommendations are available in SCP's **Findings** page when you filter for findings that Assistant suggests should be ignored, and in the [finding's details](/semgrep-code/findings/#view-findings-details-about-a-specific-finding). 
```

Comment:
```suggestion
Auto-triage recommendations are available in SCP's **Findings** page when you filter for findings that Assistant suggests should be ignored and in the [finding's details](/semgrep-code/findings/#view-findings-details-about-a-specific-finding). 
```

Reply:
Hmm, this is another one that really doesn't sound right to me without the comma. Technically yes, it's a chain of two basic clauses, but having the conditional "when you filter..." pushes the end of the first main clause a long way from the and, making it hard when you get to the and to figure out where you need to go back and break it.